elementary/config/config.py

```diff
@@ -31,6 +32,7 @@ def __init__(self, config_dir: str = DEFAULT_CONFIG_DIR, profiles_dir: str = DEF
         self.slack_webhook = slack_webhook or config.get(self._SLACK, {}).get('notification_webhook')
         self.slack_token = slack_token or config.get(self._SLACK, {}).get('token')
         self.slack_channel_name = slack_channel_name or config.get(self._SLACK, {}).get('channel_name')
+        self.slack_timezone = slack_timezone or config.get(self._SLACK, {}.get('timezone'))
```

Comment:
The timezone is part of the alert content and agnostics to the platform it is shared on.
So I think that a better name is just `timezone` 🙂 
Then we will be able to use it to affect other parts of the system if needed!

Reply:
Plus see my comments above about moving the timezone argument from the slack hierarchy to the global hierarchy of the config file 🙂